portfolio/src/main/webapp/experience.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link href='https://fonts.googleapis.com/css?family=Sora' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Be Vietnam' rel='stylesheet'>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="script.js"></script>
+  </head>
+  <style>
+  </style>
+  <body>
+    <div id="content">
+
+      <!-- navigation bar -->
+      <nav class="w3-bar w3-black">
+          <a href="index.html" class="w3-bar-item w3-button">Home</a>
+          <a href="/" class="w3-bar-item w3-button">Experience</a>
+          <a href="interest.html" class="w3-bar-item w3-button">Interest</a>
+          <a href="#" class="w3-bar-item w3-button">Gallery</a>
+          <a href="#" class="w3-bar-item w3-button">Contact</a>
+          <a class="w3-bar-item w3-right"><b>Rania Rusly</b></a>
+      </nav>
+      
+      <!-- Google-->
+      <div class="w3-cell-row work-container">
+        <div class ="w3-cell w3-center w3-mobile middle" style="width:300px"> 
+          <img class="img-circle" src="/images/google-logo.png" style="width:150px">
+        </div>
+        <div class="w3-cell w3-align-left  w3-mobile middle">
+          <div class="sub-title"><p>GOOGLE</p></div>
```

Comment:
Semantically, this is not a paragraph, right? To be semantically correct, could omit using `<p>` tag here?